### PR TITLE
Update Spine version

### PR DESCRIPTION
**Pull request by dmytro-grankin**
This PR updates Spine version to `0.9.64-SNAPSHOT`, that solves storage-related issues and improves tests for storages.

As a result, several issues were detected and fixed:
- Entity querying. Now works correctly for the `CompositeColumnFilter`, that has `ColumnFilter`s with the same name.
- `InsertEntityQuery` and `UpdateEntityQuery`. Fixed setting of entity column values in the `SQL` queries.
- Reading of events. Now the select query takes into account versions additionally to the creation time.

### CODE REVIEW

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java [hunk 1]
```diff
@@ -41,6 +48,27 @@
...
+     * Obtains the name-to-index transformer for the specified record.
+     *
+     * @param record           the record to extract column names
+     * @param firstColumnIndex the index of the first entity column in a query
+     * @return a {@code Function} transforming a column name to the its index
+     * @see io.spine.server.entity.storage.ColumnRecords#feedColumnsTo
+     */
+    Function<String, Integer> getTransformer(EntityRecordWithColumns record,
+                                             int firstColumnIndex) {
+        final Map<String, Integer> result = new HashMap<>();
```
Comment (armiol) [inline rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java]: Please use Guava for consistency with the rest of code.

Also this has to be an immutable map at the end. So I would use a builder for the `ImmutableMap`.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java [hunk 2]
```diff
@@ -41,6 +48,27 @@
...
 
+    /**
+     * Obtains the name-to-index transformer for the specified record.
+     *
+     * @param record           the record to extract column names
+     * @param firstColumnIndex the index of the first entity column in a query
+     * @return a {@code Function} transforming a column name to the its index
+     * @see io.spine.server.entity.storage.ColumnRecords#feedColumnsTo
+     */
+    Function<String, Integer> getTransformer(EntityRecordWithColumns record,
```
Comment (armiol) [inline rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java]: The name isn't specific enough.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java [hunk 3]
```diff
@@ -41,6 +48,27 @@
...
+                                             int firstColumnIndex) {
+        final Map<String, Integer> result = new HashMap<>();
+
+        int index = firstColumnIndex;
+        for (String entry : record.getColumnNames()) {
```
Comment (armiol) [inline rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java]: As of now, this list isn't sorted. So the column numbers are going to be random each time this code is executed.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/InsertAggregateRecordQuery.java [hunk 4]
```diff
@@ -70,13 +77,30 @@ protected PreparedStatement prepareStatement(ConnectionWrapper connection) {
             final Timestamp timestamp = getRecord().getTimestamp();
             statement.setLong(3, timestamp.getSeconds());
```
Comment (armiol) [inline rdbms/src/main/java/io/spine/server/storage/jdbc/query/InsertAggregateRecordQuery.java]: Please add a `todo` to get rid of this indexes. They are "magic numbers" at the moment.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/SelectEventRecordsById.java [hunk 5]
```diff
@@ -51,21 +52,24 @@
  * Query that selects {@link AggregateEventRecord} by corresponding aggregate ID sorted by
  * time descending.
  *
+ * <p>If the creation time is the same for few records, orders them by version descending.
```
Comment (armiol) [inline rdbms/src/main/java/io/spine/server/storage/jdbc/query/SelectEventRecordsById.java]: `for several records`

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java (hunk 2 above)
Comment (dmytro-grankin) [reply rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java]: Done.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java (hunk 1 above)
Comment (dmytro-grankin) [reply rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java]: Done.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/InsertAggregateRecordQuery.java (hunk 4 above)
Comment (dmytro-grankin) [reply rdbms/src/main/java/io/spine/server/storage/jdbc/query/InsertAggregateRecordQuery.java]: Done.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/SelectEventRecordsById.java (hunk 5 above)
Comment (dmytro-grankin) [reply rdbms/src/main/java/io/spine/server/storage/jdbc/query/SelectEventRecordsById.java]: Done.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java (hunk 3 above)
Comment (dmytro-grankin) [reply rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java]: There is test, `RecordStorageShould.update_entity_column_values` in `core-java`, which tests such cases.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/AggregateStorageQueryFactory.java [hunk 6]
```diff
@@ -71,9 +71,9 @@ public Logger getLogger() {
 
     /** Returns a query that selects aggregate records by ID sorted by time descending. */
     @SuppressWarnings("InstanceMethodNamingConvention")
-    public SelectByIdSortedByTimeDescQuery<I> newSelectByIdSortedByTimeDescQuery(I id) {
-        final SelectByIdSortedByTimeDescQuery.Builder<I> builder =
-                SelectByIdSortedByTimeDescQuery.<I>newBuilder(mainTableName)
+    public SelectEventRecordsById<I> newSelectByIdSortedByTimeDescQuery(I id) {
+        final SelectEventRecordsById.Builder<I> builder =
+                SelectEventRecordsById.<I>newBuilder(mainTableName)
                         .setDataSource(dataSource)
```
Comment (armiol) [inline rdbms/src/main/java/io/spine/server/storage/jdbc/query/AggregateStorageQueryFactory.java]: I think this line isn't formatted. Same with the lines below.

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java [hunk 7]
```diff
@@ -41,6 +46,30 @@
         return columnTypeRegistry;
     }
 
+    /**
+     * Obtains {@code Function} identifying entity columns of a record in a {@code SQL} query.
```
Comment (armiol) [inline rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java]:
```
Obtains a {@code Function} identifying entity columns of a record in a {@code SQL} query by the index for each column name.
...
```

File: rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java [hunk 8]
```diff
@@ -41,6 +46,30 @@
...
+     *
+     * <p>The function transforms a column name to the index in a query.
+     * The index should be used to insert column values to {@code PreparedStatement},
```
Comment (armiol) [inline rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java]:
```
Such an index may be used to insert column values to a {@code PreparedStatement} via  the {@linkplain io.spine.server.entity.storage.ColumnRecords#feedColumnsTo feedColumnsTo() utility}.
```

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

**Pull request merged**